Makefile

```diff
@@ -50,6 +50,14 @@ ALL = \
 	$(ISO_ROOT)/product.txt \
 	\
 	$(SCALITY_EL7_REPO) \
+	\
+	$(ISO_ROOT)/images/coredns.tar.gz \
```

Comment:
Let's have the version/tag number in the tarball filename.

Reply:
done